This patch fixes the tests to run on solaris.
This patch has not been contributed upstream, but is planned to be done by
 2013-Jul-12.

--- openscap-0.8.1/tests/API/SEAP/test_api_seap_parser.c.orig	2012-04-16 05:46:10.857636161 -0700
+++ openscap-0.8.1/tests/API/SEAP/test_api_seap_parser.c	2012-04-16 05:46:33.459579066 -0700
@@ -52,7 +52,7 @@
 #elif defined(__linux__) || defined(__GLIBC__)
                         getline (&input, &inlen, stdin);
 #elif defined(__SVR4) && defined(__sun)
-                        rpl_getline (&input, &inlen, stdin);
+                        getline (&input, &inlen, stdin);
 #else
 # error "FIXME"
 #endif
--- openscap-0.8.1/tests/API/SEAP/test_api_sexp_ID.c.orig	2012-04-16 05:47:12.392123448 -0700
+++ openscap-0.8.1/tests/API/SEAP/test_api_sexp_ID.c	2012-04-16 05:47:21.468821876 -0700
@@ -42,7 +42,7 @@
 #elif defined(__linux__) || defined(__GLIBC__)
                         getline (&input, &inlen, stdin);
 #elif defined(__SVR4) && defined(__sun)
-                        rpl_getline (&input, &inlen, stdin);
+                        getline (&input, &inlen, stdin);
 #else
 # error "FIXME"
 #endif
